packages/compass-crud/src/components/readonly-filter.tsx

```diff
@@ -49,21 +43,23 @@ export function ReadonlyFilter({
   queryLabel,
   filterQuery,
 }: ReadonlyFilterProps) {
+  const readOnlyFilterId = useId('readonly-filter');
   return (
-    <TextInput
-      id="readonly-filter"
-      data-testid="readonly-filter"
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore the label can be any component, but it's weirdly typed to string
-      label={
-        <QueryLabel
-          label={queryLabel}
-          tooltip="Return to the Documents tab to edit this query."
-        />
-      }
-      disabled={true}
-      value={filterQuery}
-      className={textInputStyles}
-    />
+    <>
+      <div className={queryLabelStyles}>
+        <Label htmlFor={readOnlyFilterId}>{queryLabel}</Label>
+        <InfoSprinkle align="right">
+          Return to the Documents tab to edit this query.
+        </InfoSprinkle>
+      </div>
+      <KeylineCard
+        id={readOnlyFilterId}
+        data-testid="readonly-filter"
+        disabled={true}
```

Comment:
This is probably a leftover from this being a textinput? There is no meaning in passing disabled on a div